package.json

```diff
@@ -1,18 +1,20 @@
 {
-  "name": "stellarexplorer",
-  "description": "Ledger explorer for the Stellar network",
+  "name": "pi-explorer",
```

Comment:
lots of unrelated changes for a PR titled `Turkish translation file added`..

what is pi-explorer? doesn't make sense to change the name of this project to that.

please create a separate PR with just the Turkish translation if you'd like that added to stellar explorer.

thanks.